exercises/simple-linked-list/instructions.md

```diff
@@ -0,0 +1,11 @@
+# Instructions
+
+Your task is to create a prototype for a playlist feature for a music player application.
+
+For the prototype version, create a singly linked list containing a range of numbers.
```

Comment:
Not sure if this helps, but maybe:

```suggestion
For the prototype version, create a singly linked list containing a range of numbers (the songs' IDs).
```

It might be useful to link the instructions to the theme a bit here.

Reply:
They're the song titles, not the song IDs, right?

Reply:
I kind of like the idea of linking it back, but we're using dummy data, not song data for the prototype (in my mind, anyway).